portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -636,10 +636,14 @@ definition {
 
 		FormsAdminNavigator.gotoEditFormByName(formName = "Form 1");
 
-		FormFields.viewParagraphField(
-			fieldName = "Paragraph",
-			fieldTitle = "Paragraph",
-			fieldValue = "Test");
+		WaitForElementPresent(
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#PARAGRAPH_TITLE");
+
+		WaitForElementPresent(
+			key_fieldName = "Paragraph",
+			locator1 = "FormFields#PARAGRAPH_BODY_TEXT_CONTENT",
+			value1 = "Test");
```

Comment:
@rafaellajordao, why don't you move these `WaitForElementPresent` calls into macro `viewParagraphField`?

This way we can fix this issue for every test invoking this macro. What do you think?